crates/matrix-sdk/src/room/mod.rs

```diff
@@ -17,12 +17,12 @@ use matrix_sdk_common::timeout::timeout;
 use mime::Mime;
 #[cfg(feature = "e2e-encryption")]
 use ruma::events::{
-    room::encrypted::OriginalSyncRoomEncryptedEvent, AnySyncMessageLikeEvent, AnySyncTimelineEvent,
-    SyncMessageLikeEvent,
+    marked_unread::MarkedUnreadEventContent, room::encrypted::OriginalSyncRoomEncryptedEvent,
```

Comment:
The CI is reporting an error here: `MarkedUnreadEventContent` is imported only when the feature `e2e-encryption` is enabled. We want it even if it's disabled.